### PR TITLE
DOC convert a few more references to bibtex

### PR DESCRIPTION
**Pull request by romanlutz**
## Description

Continuing with #1069. I didn't quite know whether the remaining ones at the bottom of the page should stay as is or also get converted to bibtex. I can't find existing bibtex entries for them since they're just blog posts etc.

I also just realized we don't always use `:footcite:` but sometimes additional specifications like `:footcite:p:`. If you have suggestions/thoughts please submit comments or directly commit them. @MiroDudik 

I would also love to order the bibtex file by alias, but that will have to happen in a separate PR for readability reasons.

## Tests

- [x] no new tests required
- [ ] new tests added
- [ ] existing tests adjusted

## Documentation

- [ ] no documentation changes needed
- [x] user guide added or updated
- [ ] API docs added or updated
- [ ] example notebook added or updated

### CODE REVIEW

**Comment by hildeweerts** [comment]
>  I can't find existing bibtex entries for them since they're just blog posts etc.

For the sake of consistency, we could consider adding simple `@misc` BibTeX entries for blog posts etc. with the title, URL, and date last accessed.

**Comment by romanlutz** [comment]
There are still a whole lot more references to convert, but I'd like to keep this PR at a manageable size. @fairlearn/fairlearn-maintainers can you review and let me know if it's ready to merge?

**Review by hildeweerts — APPROVED** [review]

File: docs/refs.bib [hunk 1]
```diff
@@ -184,6 +208,14 @@ @inproceedings{jacobs2021measurement
 series = {FAccT '21}
 }
 
+@misc{kain1975housing,
+  title={Housing markets and racial discrimination: A microeconomic},
```
Comment (riedgar-ms) [inline docs/refs.bib:212]: These title is feeling truncated

File: examples/plot_credit_loan_decisions.py [hunk 2]
```diff
@@ -66,17 +65,17 @@
 # same concept as anti-discrimination law. An AI system can comply with
 # anti-discrimination law while exhibiting fairness-related concerns. On the
 # other hand, some fairness interventions may be illegal under
-# anti-discrimination law. `Xiang and Raji (2019)
-# <https://arxiv.org/abs/1912.00761>`_ discuss the compatibilities and
+# anti-discrimination law. Xiang and Raji (2019)
```
Comment (riedgar-ms) [inline examples/plot_credit_loan_decisions.py:68]: This is where you want to use the different `:footcite:` modes, to automatically generate the authors and year.

File: examples/plot_credit_loan_decisions.py [hunk 3]
```diff
@@ -410,7 +407,7 @@ def resample_training_data(X_train, Y_train, A_train):
 #
 # A secondary harm that is somewhat unique to credit lending decisions is the
 # long-term impact on an individual's credit score. In the United States, a
-# `FICO credit score <https://www.investopedia.com/terms/c/credit_score.asp>`_
```
Comment (riedgar-ms) [inline examples/plot_credit_loan_decisions.py:413]: Why is this being changed to a citation? The link seems more appropriate.

File: examples/plot_credit_loan_decisions.py [hunk 4]
```diff
@@ -652,12 +648,13 @@ def plot_group_metrics_with_error_bars(metricframe, metric, error_name):
...
 # ==============================================
 # In the Fairlearn package, *postprocessing* mitigation is offered through
-# the :code:`ThresholdOptimizer` algorithm, following `Hardt, Price, and Srebro (2016)
-# <https://arxiv.org/abs/1610.02413>`_. The :code:`ThresholdOptimizer` takes in an
-# exisiting (possilbly pre-fit) machine learning model whose predictions acts as
-# a scoring function to identify separate thresholds for each *sensitive
-# feature* group. The :code:`ThresholdOptimizer` optimizes a specified objective
-# metric (in our case, :code:`balanced_accuracy`) subject to some fairness constraint
+# the :code:`ThresholdOptimizer` algorithm, following Hardt, Price, and Srebro
+# (2016) :footcite:`hardt2016equality`.
```
Comment (riedgar-ms) [inline examples/plot_credit_loan_decisions.py:652]: Again, use one of the other `:footcite:` modes to generate the author list and year automatically.

File: examples/plot_credit_loan_decisions.py [hunk 5]
```diff
@@ -1244,7 +1240,7 @@ def create_metricframe_w_errors(mframe, metrics_to_report, metric_error_pair):
 # ======================
 #
 # A key facet of Responsible Machine Learning is responsible documentation
-# practices. `Mitchell et al. (2019) <https://arxiv.org/abs/1810.03993>`_
+# practices. Mitchell et al. (2019) :footcite:`mitchell2019model`
```
Comment (riedgar-ms) [inline examples/plot_credit_loan_decisions.py:1243]: Another case for having BibTeX do what it's supposed to do....

File: examples/plot_credit_loan_decisions.py [hunk 6]
```diff
@@ -1263,6 +1259,12 @@ def create_metricframe_w_errors(mframe, metrics_to_report, metric_error_pair):
 # practitoners may run into the issue of not having access to the sensitive
 # demographic features. As a result, financial institutions are often tasked
 # with proving their models are compliant with fair lending laws by imputing
-# these demographics features. However, `Chen et. al (2018)
-# <https://arxiv.org/abs/1811.11154>`_ show these imputation methods often
+# these demographics features. However, Chen et. al (2018)
```
Comment (riedgar-ms) [inline examples/plot_credit_loan_decisions.py:1262]: And another.

File: docs/refs.bib [hunk 7]
```diff
@@ -285,4 +317,127 @@ @inproceedings{barocas2017problem
...
+@misc{scikitlearn2022ames,
+  title={The Ames housing dataset},
+  author={Scikit-Learn developers},
+  year={2022},
+  url={https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_ames_housing.html}
+}
+
+@misc{scikitlego2019fairness,
+  title={Fairness},
+  author={Vincent Warmerdam, Matthijs Brouns, and Scikit-Lego contributors},
```
Comment (riedgar-ms) [inline docs/refs.bib:350]: I'm not sure this is how BibTeX expects the author list to be formatted. See some of the other entries

**Review by riedgar-ms — CHANGES_REQUESTED** [review]
I'm not sure all these citation conversions are appropriate, and the other `:footcite:` modes should be being used when "Author (year)" citations are in use.

File: docs/refs.bib (hunk 1 above)
Comment (romanlutz) [reply docs/refs.bib:212]: bibtex file was bad, fixed it now

**Comment by romanlutz** [comment]
> I'm not sure all these citation conversions are appropriate, and the other `:footcite:` modes should be being used when "Author (year)" citations are in use.

I've addressed all your comments (I think). Please take another look whenever you have time 🙂

File: docs/user_guide/datasets/boston_housing_data.rst [hunk 8]
```diff
@@ -402,47 +412,17 @@ different types of statistical analyses and different predicted outcomes.
...
+References
+----------
 
-  .. [#10] `"Metropolitan Areas", <https://www.census.gov/history/www/programs/geography/metropolitan_areas.html>`_,
-        United States Census Bureau.
+.. footbibliography::
 
-  .. [#11] Kinmberlé Crenshaw, Mapping the margins: Intersectionality, identity politics, and violence against women of color,
-      Stanford Law Review, 1993, 43(6), 1241-1299.
+.. [#1] Marshall Hargrave, `"Hedonic Pricing" <https://www.investopedia.com/terms/h/hedonicpricing.asp>`_,
```
Comment (riedgar-ms) [inline docs/user_guide/datasets/boston_housing_data.rst:427]: Why not just leave the link in the run of the text? I'm not sure why this is being drawn out as a specific citation.

**Review by riedgar-ms — APPROVED** [review]

File: docs/user_guide/datasets/boston_housing_data.rst (hunk 8 above)
Comment (romanlutz) [reply docs/user_guide/datasets/boston_housing_data.rst:427]: I must have been absent-minded... that's what I wanted to revert actually... I totally agree

**Pull request merged**